tqdm/_tqdm.py

```diff
@@ -402,8 +402,14 @@ def inner(df, func, *args, **kwargs):
                 # Precompute total iterations
                 total = getattr(df, 'ngroups', None)
                 if total is None:  # not grouped
-                    total = len(df) if isinstance(df, Series) \
-                        else df.size // len(df)
+                    if isinstance(df, Series):
+                        total = len(df)
+                    elif kwargs.get('axis', 0) == 0:
+                        # over columns
+                        total = df.size // len(df)
+                    else:
+                        # over rows
```

Comment:
Neither did I XD Yes there must more axes but it's just the same argument I think, so it's the same management (just use `panel.shape[x]` where x is the axis if I'm not mistaken).